### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ColorContrast: The color contrast of the tab focus on link 1 is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12963]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#Benchmark;#ColorContrast;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu
4. Tab to link 1
5. Open accessibility insights for desktop and check the ratio of the tab focus outline on link 1

**Actual:**
On navigating to link 1 with tab and checking the color contrast of the focus on link1 the ratio is 2.122:1.

**Same issue observed in page:**

1. Import Extension - Preview Data
2. Import Extension - Modify Columns
3. Import Extension - Summary

**Expected:**
The focus ratio on link 1 should be 3:1 or greater.

**User Impact:**
Low vision users will not be able to view the focus.

**Attachment:**
![#12963_A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ColorContrast](https://user-images.githubusercontent.com/24580570/96403408-f3cead00-11f5-11eb-9456-b1a2d0ccf3ab.png)

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version: 1.42.0-insider(user-setup)

![12963_issue fixed](https://user-images.githubusercontent.com/93374043/218951383-925c4cc9-e2db-40bc-83f3-474d113cb315.png)

**GithubTags**:#Closed;
